spaceship/lib/spaceship/tunes/build_train.rb

```diff
@@ -98,7 +123,7 @@ def update_testing_status!(new_value, testing_type, build = nil)
 
             # also update the builds
             train['builds'].delete_if do |b|
-              return true if b[testing_key].nil?
+              next true if b[testing_key].nil?
```

Comment:
`next true`? Is that being used in the `delete_if`? If so, this might be a bit confusing, we should maybe refactor it to be a multi-line if else statement